app/controllers/api/v2/ext_app/containers_controller.rb

```diff
@@ -50,6 +50,19 @@ def rebootstrap
     render json: ::Api::V2::ExtApp::ContainerSerializer.new(@container).to_h
   end
 
+  # POST /schedule_relocation
+  def schedule_relocation
+    @cluster = ::Cluster.find_by!(name: params[:cluster_name])
+    node = ::Node.find_by!(hostname: params[:node_hostname]) 
+    @container = @cluster.containers.exists.find_by!(hostname: params[:hostname], status: ["BOOTSTRAPPED", "BOOTSTRAP_ERROR"])
+    if @container.node_id == node.id
+      render json: { success: false, errors: "can't relocate container to same node", code: 400 }, status: :bad_request
+      return
+    end
+    @container.update(status: "SCHEDULE_RELOCATION", node_id: node.id)
```

Comment:
why don't we use `update_status` method here? similar to flow in other controller for relocation

Reply:
because we need update node_id also